spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/json-shade/java/org/springframework/boot/configurationprocessor/json/JSONArray.java

```diff
@@ -65,8 +65,8 @@ public JSONArray() {
 	public JSONArray(Collection copyFrom) {
 		this();
 		if (copyFrom != null) {
-			for (Iterator it = copyFrom.iterator(); it.hasNext();) {
-				put(JSONObject.wrap(it.next()));
+			for (Object o : copyFrom) {
```

Comment:
```suggestion
			for (Object it : copyFrom) {
```

尽量不要用单一字符`o`做变量名，容易和0混淆。